presto-docs/src/main/sphinx/sql/explain-analyze.rst

```diff
@@ -0,0 +1,58 @@
+===============
+EXPLAIN ANALYZE
+===============
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    EXPLAIN ANALYZE statement
+
+Description
+-----------
+
+Execute the statement and show the distributed execution plan of the statement
+along with the cost of each operation.
+
+.. note::
+
+    The stats may not be entirely accurate, especially for queries that complete quickly.
+
+Examples
+--------
+
+In the example below, you can see the CPU time spent in each stage, as well as the relative
+cost of each operator in the stage. Note that the relative cost of the operators is based on
+wall time, which may or may not be correlated to CPU time.
+
+.. code-block:: none
+
+    presto:sf1> EXPLAIN ANALYZE SELECT count(*), clerk FROM orders GROUP BY clerk;
```

Comment:
why won't you use ` EXPLAIN ANALYZE SELECT regionkey, count(*) FROM nation GROUP BY 1` to be consistent with `explain.rst`?